rfcs/0008-commitstatus/README.md

```diff
@@ -0,0 +1,71 @@
+
+# RFC-xxxx Create a new API kind for Commit Status
+
+**Status:** provisional
+
+**Creation date:** 2022-05-12
+
+**Last update:** 2022-05-12
+
+## Summary
+
+There should be a dedicated kind in the notification controller for sending commit status notifications to git providers.
+
+## Motivation
+
+Currently, The Alert type can reference both git providers and chat providers. However, the differences between the two providers and how notifications being sent to them should be handled has continued to diverge. 
+For example, there is a limit on the length of the name/context of the status for git provider.
+
+When one commit triggers more than one deployment/reconciliation in different clusters, the commit status notification from one overwrites the other as seen in this [issue](https://github.com/fluxcd/notification-controller/issues/195).
+A new field specific to the git commit status will allow Flux users to specify a prefix that will be prepended to the context to differentiate the statuses.
+
+By creating a separate kind for git commit status, the specific nuances of the provider can be properly taken into account and it would allow more flexibility in adding fields specific to each kind.
```

Comment:
I would restructure the argumentation here to gradually get to the point of introducing a new kind, because the changes to the current kind or not justified.

Paraphrasing:

```
We have observed the issue to be <problem>, this would require the introduction of
<something> because of <reason>. Which is ultimately solved by <introducing new kind>,
because of <argumentation>.
```